config/default.yml

```
@@ -129,6 +129,12 @@ Rails/ActiveSupportAliases:
   Enabled: true
   VersionAdded: '0.48'
 
+Rails/ActiveSupportOnLoad:
+  Description: 'Use `ActiveSupport.on_load(...)` to patch Rails core classes.'
+  StyleGuide: 'https://rails.rubystyle.guide/#hash-conditions'
+  Enabled: 'pending'
```

I think the behavior is incompatible.

```suggestion
  SafeAutoCorrect: false
  Enabled: 'pending'
```